addons/test_base_automation/tests/test_flow.py

```diff
@@ -1199,6 +1199,30 @@ def test_automation_form_view_time_triggers(self):
         automation = automation_form.save()
         self.assertEqual(automation.filter_pre_domain, False)
 
+    def test_automation_form_view_with_default_values_in_context(self):
+        # Use case where default model, trigger and trigger_fields in context
+        context = {
+            'default_name': 'Test Automation',
+            'default_model_id': self.env.ref('test_base_automation.model_base_automation_lead_test'),
+            'default_trigger': 'on_create_or_write',
+            'default_trigger_field_ids': self.env.ref('test_base_automation.field_base_automation_lead_test__state'),
```

Comment:
I'm guessing defaults cannot contain recordsets add .ids at the end

And adapt the asserts accordingly